### PR TITLE
New Indian Spices Cheat Sheet

### PR DESCRIPTION
**Pull request by shahbaz17**
https://duck.co/ia/view/indian_spices_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by shahbaz17** [comment]
@moollaza Please review this cheat sheet.

**Comment by GuiltyDolphin** [comment]
@shahbaz17 It looks like the tests have failed! Please look at the Travis CI log and correct the issues mentioned.

**Comment by shahbaz17** [comment]
Everything look perfect now. Need review
@moollaza @GuiltyDolphin

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=hindi spice name cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/indian_spices_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=hindi spice name cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/indian_spices_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/indian-spices.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
+{
+    "id" : "indian_spices_cheat_sheet",
+    "name" : "Indian Spices Names",
+    "description" : "Indian spices include a variety of spices grown across the Indian subcontinent. Find hindi names of Indian spices.",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/indian-spices.json]: Let's simplify this to `"English and Hindi names for a variety of spices grown across the Indian subcontinent"`

File: share/goodie/cheat_sheets/json/indian-spices.json [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    ],
+
+    "template_type" : "language",
+
+    "section_order" : [
+        "Spices"
+    ],
+
+    "sections" : {
+        "Spices" : [
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/indian-spices.json]: To maximize space it would be best to split this into 2+ sections

**Comment by moollaza** [comment]
@shahbaz17 Thanks! This LGTM :+1:

**Comment by shahbaz17** [comment]
Thanks @moollaza and @GuiltyDolphin for guiding me.
Looking forward to contribute more to DDG

File: share/goodie/cheat_sheets/json/indian-spices.json (hunk 2 above)
Comment (shahbaz17) [inline share/goodie/cheat_sheets/json/indian-spices.json]: Sure will do it in next commit

**Comment by shahbaz17** [comment]
@moollaza https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2720 with all the above changes

**Pull request merged**